edx_sga/utils.py

```diff
@@ -19,3 +22,16 @@ def is_finalized_submission(submission_data):
     if submission_data and submission_data.get('answer') is not None:
         return submission_data['answer'].get('finalized', True)
     return False
+
+
+def get_file_modified_time_utc(file_path):
+    tz = (
+        pytz.timezone(settings.TIME_ZONE)
```

Comment:
This might not reflect an accurate value for the timestamp if the Django settings are, for example, `America/New_York` but the system time zone is set to UTC.

Reply:
ah, of course. will change